xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -6597,9 +6595,18 @@ notify(xlator_t *this, int32_t event, void *data, ...)
         }
 
         default:
+            /* Set the event_graph to false so that event
+               debug msg would not try to access invalid graph->id
+               while data object is not matched to graph object
+               for ex in case of upcall event data object represents
+               gf_upcall object
+            */
+            event_graph = _gf_false;
             break;
     }
 
+    gf_log("fuse", GF_LOG_DEBUG, "got event %d on graph %d", event,
+           ((graph && event_graph) ? graph->id : 0));
```

Comment:
Since '0' is a valid graph id, when the graph is not known I would use a different value (maybe '-1') or write different log messages depending on `event_graph` value.

Reply:
done